view/theme/frio/js/textedit.js

```diff
@@ -2,6 +2,46 @@
  * @brief The file contains functions for text editing and commenting
  */
 
+function commentGetLink(id) {
+    reply = prompt("Please enter a link URL:");
```

Comment:
"Please enter a link URL:" => "Please enter a video/audio/webpage URL:"

This string should be translated, but getting Javascript to access the translated string may prove tricky.